Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -160,6 +160,7 @@ struct Scene_polyhedron_item_priv{
   mutable QOpenGLShaderProgram *program;
   unsigned int number_of_null_length_edges;
   unsigned int number_of_degenerated_faces;
+  double genus;
```

Comment:
I do not really understand why the `genus` is a `double`. It should be an integral type.